src/smoltcp_phy.rs

```diff
@@ -1,45 +1,43 @@
 use crate::{
     EthController, rx, tx
 };
-use core::intrinsics::transmute;
+use core::{
+    cell,
+    intrinsics::transmute
```

Comment:
`core::intrinsics::transmute` still used?

Reply:
Fixed with 669fe7ab9be7591dbd4f0fba243f14a00acf8fae.

Reply:
Forced-pushed branch to fix some messy code and add delays (see #3).